Cargo.toml

```diff
@@ -46,6 +48,10 @@ gpu = ["ocl", "itertools", "fs2"]
 groth16 = ["paired"]
 multicore = ["futures-cpupool", "crossbeam", "num_cpus"]
 
+[[bin]]
```

Comment:
If we mark it as bin it will be shipped with releases on crates.io which is not really what we want. Best is probably to make it its own subcrate that doesn't get released, or put it into the `examples` folder